grouch/ui.py

```diff
@@ -2,7 +2,7 @@
 import logging
 import sys
 from textwrap import TextWrapper
-
+#from bs4 import BeautifulSoup
```

Comment:
That was accidentally left in, I was testing the ui.py script and was getting an error about missing the BeautifulSoup module. I didn't realize later that it was throwing the error for a different file.